docs/examples/nyc_taxi.ipynb

```diff
@@ -51,7 +51,7 @@
    "source": [
     "## Install dependencies\n",
     "\n",
-    "This example has been verified in Python 3.7 with the following libraries.\n",
+    "This example has been verified in Python 3.7/3.8/3.9 with the following libraries.\n",
```

Comment:
Would it be simpler to avoid listing the supported python versions so that we don't need to update this sentence every time we update the supported python version?

Reply:
I think we can avoid listing the supported Python version in the docs. But we cannot avoid listing them in the bash code, which also needs to be updated every time when we update the supported Python version, because Colab runs Python 3.10 by default and we need to install the compatible Python versions to run the notebook in Colab.